.yamato/sg2-tests.yml

```diff
@@ -0,0 +1,57 @@
+ShaderGraph2_Win_pr_trigger:
+  name: ShaderGraph2 - PR Tests Trigger
+  triggers:
+    expression: pull_request.source match "(?:^|.+\/)sg2\/.+" AND NOT pull_request.draft # this trigger the job on pull request exclusively on specific branches
+    cancel_old_ci: true
+  dependencies:
+    - .yamato/sg2-tests.yml#ShaderGraph2_Mac
+    - .yamato/sg2-tests.yml#ShaderGraph2_Win
+ShaderGraph2_Win:
+  name: ShaderGraph2 Tests - Win
+  agent:
+    type: Unity::VM
+    image: sdet/gamecode_win10:stable
+    flavor: b1.large
+  variables:
+    UTR_VERSION: "current"
+    TEST_FILTER: .*
+    EDITOR_BRANCH: "sg2/2023-01-05-working-editor"
+  commands:
+    -  command: NetSh Advfirewall set allprofiles state off
+    -  command: curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
+       retries: 2
+    -  command: choco install unity-downloader-cli -y -s https://artifactory.prd.it.unity3d.com/artifactory/api/nuget/unity-choco-local
+       retries: 2
+    -  command: unity-downloader-cli -u gtf/staging -c editor --fast
```

Comment:
You are not getting"sg2/2023-01-05-working-editor".

Reply:
Updated and moved into variable.